plugin/tools.py

```diff
@@ -342,7 +372,7 @@ def seconds_from_string(time_str):
         return int(h) * 3600 + int(m) * 60 + int(s)
 
     @staticmethod
-    def get_position_status(point, view, settings):
+    def get_pos_status(point, view, settings):
```

Comment:
I have abbreviations. :) They are even forbidden in the code base where I work. We follow this style guide: https://google.github.io/styleguide/cppguide.html#General_Naming_Rules